src/applications/vaos/tests/services/appointment/index.v2.unit.spec.js

```diff
@@ -926,11 +926,11 @@ describe('VAOS Appointment service', () => {
         [
           { op: 'remove', path: ['description'] },
           { op: 'remove', path: ['practitioners'] },
-          {
-            op: 'replace',
-            path: ['created'],
-            value: moment().format('YYYY-MM-DD'),
-          },
+          // {
+          //   op: 'replace',
+          //   path: ['created'],
+          //   value: moment().format('YYYY-MM-DD'),
+          // },
```

Comment:
Same as above. v0 and v2 now contains identical date format for the `created` attribute

Reply:
Done